scripts/validate-newissue-wrap.sh

```
@@ -0,0 +1,103 @@
+
+if ! command -v tmux >/dev/null 2>&1; then
+  echo "SKIP: tmux not installed"
+  exit 0
+fi
+
+cleanup() {
+  if [ -n "${TMUX_PID:-}" ]; then
+    tmux kill-session -t "$SESSION" 2>/dev/null || true
+  fi
+  [ -n "$CAPTURE" ] && rm -f "$CAPTURE"
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)